src/puppeteer/README.md

```diff
@@ -65,6 +65,21 @@ The server provides access to two types of resources:
 ## Configuration to use Puppeteer Server
 Here's the Claude Desktop configuration to use the Puppeter server:
 
+### Docker
+
+```json
```

Comment:
Lets add a note about the difference between implementations: Something like 
```suggestion
### Docker

**NOTE** The docker implementation will use headless chromium, where as the NPX version will open a browser window.

```json
```